pkg/util/escape.go

```diff
@@ -20,6 +20,19 @@ import (
 	"strings"
 )
 
+// EscapeQualifiedPluginName converts a plugin name in the format
+// vendor/pluginname into a proper ondisk vendor~pluginname plugin directory
+// format.
+func EscapePluginName(in string) string {
```

Comment:
After initial comments, I am trying to make this one closer to the network plugin model we have. So added this one.